package.json

```diff
@@ -31,7 +31,8 @@
 	],
 	"dependencies": {
 		"normalize-url": "^4.3.0",
-		"url-regex": "^5.0.0"
+		"re2": "^1.15.4",
```

Comment:
`re2` is already a dependency of `url-regex-safe`.

Reply:
Removed in latest commit